src/locale/misc/views/en.yml

```diff
@@ -7,6 +7,9 @@ columnDialog:
     personFields:
       description: Add your choice of fields like email, phone number, etc.
       title: Person fields
+    tag:
+      description: Show a checkmark if the person has this tag
+      title: Tag
```

Comment:
I think this text can be improved, but we can take a look at all the texts for all the choices in one go when they're all done, so that there's a consistent tone across all of them.

Reply:
(This requires no action for now)